src/plugins/intel_cpu/tests/functional/subgraph_tests/src/matmul_weights_decompression.cpp

```diff
@@ -360,7 +387,7 @@ INSTANTIATE_TEST_SUITE_P(smoke_MatMulCompressedWeights_amx,
 const std::vector<ShapeParams> input_shapes_corner_cases_basic = {
     {{{-1, -1, -1}, {{1, 4, 16}}}, {1, 16, 32}},
     {{{-1, -1, -1}, {{1, 4, 16}}}, {16, 32}},
-    {{{-1, -1, -1}, {{1, 4, 16}}}, {16, 32}, 4ul},
+    {{{-1, -1, -1}, {{1, 5, 16}}}, {16, 32}, 4ul},
```

Comment:
I'm just wondering why we changed this test case?

Reply:
Just to trigger new decompression algo which has condition on that dimension